packages/oas-utils/vitest.config.ts

```diff
@@ -0,0 +1,14 @@
+import { fileURLToPath } from 'node:url'
+import { defineConfig, mergeConfig } from 'vitest/config'
+
+import viteConfig from './vite.config'
+
+export default mergeConfig(
+  viteConfig,
+  defineConfig({
+    test: {
+      environment: 'jsdom',
+      root: fileURLToPath(new URL('./', import.meta.url)),
+    },
```

Comment:
we can just add those lines to the vite.config.ts, isn’t that easier/cleaner?

Reply:
sure, just copied this out of one of the other repos